### DISCUSSION TITLE
My station in North Vancouver, Canada

### DISCUSSION THREAD

**Discussion by mdkendall** [jvde-github/AIS-catcher#182] [Show and tell]
Here are some photos of my AIS-catcher based receiver:

<img src="https://github.com/jvde-github/AIS-catcher/assets/667844/66964f30-4705-440f-b455-088fbde2f9b1" width="34%">
<img src="https://github.com/jvde-github/AIS-catcher/assets/667844/0df59445-6eed-4778-ace3-c158ce4a2b04" width="45%">
<img src="https://github.com/jvde-github/AIS-catcher/assets/667844/923cb535-da6d-467c-8bff-ffcabf45efff" width="45%">
<img src="https://github.com/jvde-github/AIS-catcher/assets/667844/1f42e3f8-5737-4306-8c83-355a77f950e5" width="45%">

The parts list is roughly:
- Raspberry Pi Zero W
- RTL-SDR Blog V3 dongle
- Generic 128x64 0.96" OLED module with I2C interface
- Hammond 1554JL enclosure with Heyco 3582VB vent
- Connectors and cables to suit

Everything is mounted in the enclosure on a 3D printed plate which has integrated standoffs with threaded inserts for mounting the Raspberry Pi and OLED. Power is supplied at 5V via an M8 connector, and the antenna is connected via SMA.

Here it is installed together with a dipole antenna on my balcony.

<img src="https://github.com/jvde-github/AIS-catcher/assets/667844/a307ac3e-fdde-40f4-a447-80eb29002fa5" width="34%"> 
<img src="https://github.com/jvde-github/AIS-catcher/assets/667844/5f970376-53cf-44b0-ac14-283601daf1dc" width="34%">

AIS-catcher is installed and configured like this:
https://gist.github.com/mdkendall/06e29787549be47c7da409a61ed39d34

The OLED is driven like this:
https://gist.github.com/mdkendall/a43b203ff42a429706e819179b4aa80c

The receiver is located in North Vancouver, Canada, about 400m from the water's edge and elevated about 50m. It is currently receiving about 400 messages per minute and I see about 250 ships in range on the map, with most being within 10km and the furthest being about 40km.

The receiver reports to Marine Traffic here:
https://www.marinetraffic.com/en/ais/details/stations/29475

I would be happy to answer any questions.

**Comment by jvde-github** [comment]
Nice, I have added a link in the README.

**Comment by michaelgeorge001** [comment]
Very nice job.

Do you happen to have a detailed parts list available? I'm especially interested in that power connector that is mounted on the case.

**Reply by mdkendall** [reply]
The power plug is an "M8" connector. Specifically it is an A-coded, 3-pin, M8 connector. These, along with the larger M12 versions, are common on industrial sensors and switches such as inductive proximity sensors. I bought mine from AliExpress from this seller: https://www.aliexpress.com/item/32839974263.html

There is a standard wiring for these connectors. My notes say this:

![Screenshot 2023-10-16 114401](https://github.com/jvde-github/AIS-catcher/assets/667844/800d9b94-921f-4e9b-bb16-e1e3b891e361)

**Comment by roglow** [comment]
Excellent work and very helpful description.

Can you provide more detail on what antenna you are using?

**Reply by mdkendall** [reply]
It's a DIY dipole made from RG58 coax inside a three foot length of 1/4in (inner) PEX water pipe. There is a Laird 28B0591-200 ferrite near the bottom acting as a balun.

I have some notes on how I made it, and measurements of its performance, but I haven't written them up anywhere, partly because the bottom part is 3D printed and is fairly specific to mounting on my balcony railings.

**Reply by roglow** [reply]
Cool. A couple years ago, I built a J-pole antenna tuned for 2m amateur radio (144-148MHz) in the ham radio class I was taking.
